README.md

```diff
@@ -1,159 +1,59 @@
-<img src=".erb/img/erb-banner.svg" width="100%" />
+## Draft-Craft
 
-<br>
+This project is a notes app built using React and Electron. The app features a simple, user-friendly interface designed for productivity and note management.
 
-<p>
-  Electron React Boilerplate uses <a href="https://electron.atom.io/">Electron</a>, <a href="https://facebook.github.io/react/">React</a>, <a href="https://github.com/reactjs/react-router">React Router</a>, <a href="https://webpack.js.org/">Webpack</a> and <a href="https://www.npmjs.com/package/react-refresh">React Fast Refresh</a>.
-</p>
+## Features
 
-<br>
+- **Markdown-based Note-Taking**: Create, edit, and save notes using markdown.
+- **File Organization**: Organize notes into folders and subfolders.
 
-<div align="center">
+## Prerequisites
 
-[![Build Status][github-actions-status]][github-actions-url]
-[![Github Tag][github-tag-image]][github-tag-url]
-[![Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/Fjy3vfgy5q)
+Before starting, ensure that you have the following tools installed:
 
-[![OpenCollective](https://opencollective.com/electron-react-boilerplate-594/backers/badge.svg)](#backers)
-[![OpenCollective](https://opencollective.com/electron-react-boilerplate-594/sponsors/badge.svg)](#sponsors)
-[![StackOverflow][stackoverflow-img]][stackoverflow-url]
+1. **Node.js** (v20)
 
-</div>
+   - [Official Download Page](https://nodejs.org/en/download/)
+   - Installation Instructions: [How to Install Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
 
-## Install
+2. **Git**
+   - [Official Download Page](https://git-scm.com/downloads)
+   - Installation Instructions: [How to Install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 
-Clone the repo and install dependencies:
+## Installation Steps
+
+- First, clone the repository and install the necessary dependencies:
 
 ```bash
-git clone --depth 1 --branch main https://github.com/electron-react-boilerplate/electron-react-boilerplate.git your-project-name
-cd your-project-name
+git clone https://github.com/Kicchu02/Draft-Craft.git
+cd Draft-Craft
```

Comment:
Remove the clone part. We discussed this as well [here](https://github.com/Kicchu02/Draft-Craft/pull/1#issuecomment-2395024189)